sync_first/sync_first_app/templates/new_incident.html

```diff
@@ -0,0 +1,37 @@
+{% extends "base.html" %}
+{% block content %}
+<h1>הזנת אירוע</h1>
+<div class="container">

```

Comment:
Are you sure we want this form in English?

Reply:
no, it will be changed to Hebrew later.